.github/workflows/ci.yaml

```diff
@@ -8,38 +8,38 @@ name: rosetta-wallet
 
 jobs:
   rustfmt:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
     steps:
     - name: Checkout sources
       uses: actions/checkout@v3
 
     - name: Install rust toolchain
-      uses: hecrj/setup-rust-action@v1
+      uses: actions-rs/toolchain@v1
       with:
-        components: rustfmt
-
-    - name: dprint check
-      uses: dprint/check@v2.2
```

Comment:
interesting, why dprint is not working? once this is cargo tool, it is possible to install this way:
```bash
cargo install --locked dprint
```

Reply:
Actually don't know, look at this build run:
https://github.com/Analog-Labs/chain-connectors/actions/runs/5750124794/job/15586261879

Okay, let me try with `cargo`

Reply:
Added, should be good to go since we have the `dprint.json` in the repo